src/Data/List/Fresh/Relation/Unary/Any/Properties.agda

```diff
@@ -58,7 +58,7 @@ module _ {R : Rel A r} {P : Pred A p} {Q : Pred A q} (P? : Decidable P) where
   ¬All⇒Any : {xs : List# A R} → ¬ (All P xs) → Any (∁ P) xs
   ¬All⇒Any {xs = []}      ¬ps = ⊥-elim (¬ps [])
   ¬All⇒Any {xs = x ∷# xs} ¬ps with P? x
-  ... |  true because  [p] = there (¬All⇒Any (¬ps ∘′ (invert [p] ∷_)))
+  ... | true because  [p] = there (¬All⇒Any (¬ps ∘′ (invert [p] ∷_)))
```

Comment:
`true` and `false` are of different lengths, so *some* alignment is required if we want (the usually desirable?) alignment of the ` = definiens` part of each `with` clause. What's the right thing to do? Not the change you've made as it stands, i think.